app/javascript/validate-paired-tubes/components/ValidatePairedTubes.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <lb-page>
+    <lb-loading-modal
+      v-if="loading"
+      :message="progressMessage"
+    />
+    <lb-main-content>
+      <lb-transfer-volumes
+        :purpose-configs="purposeConfigs"
+        :tube="sourceTube.labware"
+        :confirmed-pair="allValid"
+      />
+    </lb-main-content>
+    <lb-sidebar>
+      <b-card
+        header="Validate Tubes"
+        header-tag="h3"
+      >
+        <p class="tv-instructions">
+          Scan the LTHR-384 Pool XP tube and the corresponding LB Lib Pool Norm tube. If they match, the transfer volumes will be shown to the left.
+        </p>
+        <div class="tube-scan-fields">
+          <lb-labware-scan
+            key="source"
+            ref="sourceScan"
+            :api="devourApi"
+            label="Pool XP"
```

Comment:
Ahh, the hard-coded names here are a little more useful, but source and destination could also be used?